pkgs/top-level/all-packages.nix

```diff
@@ -11954,6 +11954,8 @@ with pkgs;
 
   hiawatha = callPackage ../servers/http/hiawatha {};
 
+  home-assistant = python3Packages.callPackage ../servers/home-assistant { };
```

Comment:
This should simply be `callPackage` because we do not have any Python packages to supply as arguments.

Reply:
That's true, but it will hopefully change in the future when home-assistant allows newer versions of the dependencies :)

Reply:
If we have packages from `python-packages.nix` and packages defined locally (read: the referred file) then we need to update the Python package set with `packageOverrides`. There will not be any use for defining Python packages as parameters, and as such no case for using `pythonPackages.callPackage`.

Reply:
I know, but I hope that home-assistant will just work with the standard python package set in the future.

Reply:
Currently the not overridden Python package set (see arguments `python` and `isPy3k`) is mixed with the overridden set. That should not happen. 

> I know, but I hope that home-assistant will just work with the standard python package set in the future.

Very unlikely. We have a lot of Python applications, and for the larger ones we typically need to provide overrides.